packaging/windows/WindowsLauncher.cs.in

```diff
@@ -11,15 +11,126 @@
 
 using System;
 using System.Diagnostics;
-using System.Drawing;
 using System.IO;
 using System.Linq;
 using System.Media;
 using System.Reflection;
-using System.Windows.Forms;
+using System.Runtime.InteropServices;
+using System.Text;
 
 namespace OpenRA
 {
+	// Minimal SDL wrapper to work around SDL2-CS requiring netstandard2.0
```

Comment:
If we want to go ahead with this approach I will need to reword this to clearly attribute SDL2-CS and its license.